extensions/roc-package-web-app-react/app/shared/flux/create-store.js

```diff
@@ -50,11 +51,13 @@ export default function createReduxStore(reducers, ...middlewares) {
                 finalCreateStore = compose(
                     applyMiddleware(...normalMiddlewares, ...debugMiddlewares),
                     devTools,
-                    persistState(window.location.href.match(/[?&]debug_session=([^&]+)\b/))
+                    persistState(window.location.href.match(/[?&]debug_session=([^&]+)\b/)),
+                    ...enhancers
                 )(createStore);
             } else {
                 finalCreateStore = compose(
-                    applyMiddleware(...normalMiddlewares)
+                    applyMiddleware(...normalMiddlewares),
+                    ...enhancers
```

Comment:
That is a really good question that I do not have a good answer for. It seems that most enhancers are interchangeable in the order that they are added and all I have seen are added after middlewares.